### PR TITLE
Generated  models and request builders

### PR DESCRIPTION
**Pull request by andrueastman**
This pull request was automatically created by Azure Pipelines. **Important** Check for unexpected deletions or changes in this PR.

### CODE REVIEW

File: src/main/java/com/microsoft/graph/generated/identity/authenticationeventsflows/item/graphexternalusersselfservicesignupeventsflow/onauthenticationmethodloadstart/graphonauthenticationmethodloadstartexternalusersselfservicesignup/GraphOnAuthenticationMethodLoadStartExternalUsersSelfServiceSignUpRequestBuilder.java [hunk 1]
```diff
@@ -45,37 +45,37 @@ public GraphOnAuthenticationMethodLoadStartExternalUsersSelfServiceSignUpRequest
...
-     * Get the items of type microsoft.graph.onAuthenticationMethodLoadStartExternalUsersSelfServiceSignUp in the microsoft.graph.onAuthenticationMethodLoadStartHandler collection
+     * Get the item of type microsoft.graph.onAuthenticationMethodLoadStartHandler as microsoft.graph.onAuthenticationMethodLoadStartExternalUsersSelfServiceSignUp
      * @param requestConfiguration Configuration for the request such as headers, query parameters, and middleware options.
-     * @return a {@link InputStream}
+     * @return a {@link OnAuthenticationMethodLoadStartExternalUsersSelfServiceSignUp}
      * @throws ODataError When receiving a 4XX or 5XX status code
      */
     @jakarta.annotation.Nullable
-    public InputStream get(@jakarta.annotation.Nullable final java.util.function.Consumer<GetRequestConfiguration> requestConfiguration) {
+    public OnAuthenticationMethodLoadStartExternalUsersSelfServiceSignUp get(@jakarta.annotation.Nullable final java.util.function.Consumer<GetRequestConfiguration> requestConfiguration) {
```
Comment (Ndiritu) [inline src/main/java/com/microsoft/graph/generated/identity/authenticationeventsflows/item/graphexternalusersselfservicesignupeventsflow/onauthenticationmethodloadstart/graphonauthenticationmethodloadstartexternalusersselfservicesignup/GraphOnAuthenticationMethodLoadStartExternalUsersSelfServiceSignUpRequestBuilder.java:63]: Breaking change.

File: src/main/java/com/microsoft/graph/generated/invitations/InvitationsRequestBuilder.java [hunk 2]
```diff
@@ -32,16 +33,20 @@ public CountRequestBuilder count() {
...
     }
     /**
-     * Provides operations to manage the collection of invitation entities.
-     * @param invitationId The unique identifier of invitation
-     * @return a {@link InvitationItemRequestBuilder}
+     * Provides operations to manage the invitedUser property of the microsoft.graph.invitation entity.
+     * @return a {@link InvitedUserRequestBuilder}
      */
     @jakarta.annotation.Nonnull
-    public InvitationItemRequestBuilder byInvitationId(@jakarta.annotation.Nonnull final String invitationId) {
```
Comment (Ndiritu) [inline src/main/java/com/microsoft/graph/generated/invitations/InvitationsRequestBuilder.java:40]: Deprecated method. Breaking change

File: src/main/java/com/microsoft/graph/generated/invitations/item/InvitationItemRequestBuilder.java [hunk 3]
```diff
@@ -1,246 +0,0 @@
-package com.microsoft.graph.invitations.item;
```
Comment (Ndiritu) [inline src/main/java/com/microsoft/graph/generated/invitations/item/InvitationItemRequestBuilder.java:1]: Deleted request builder. Breaking.

**Review by Ndiritu — CHANGES_REQUESTED** [review]

**Comment by baywet** [comment]
superseded by #2085

**Pull request closed without merging**